CMakeLists.txt

```diff
@@ -3,7 +3,24 @@ project(openGA CXX C)
 set(CMAKE_CXX_STANDARD 11)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
-find_package(Boost COMPONENTS system filesystem thread chrono REQUIRED)
+# Eigen
+include_directories("/usr/include/eigen3" )
+#TdLib
+set(TdLib_DIR "/home/wang/CLionProjects/TdLibrary/install/lib/CMake/TdLib/")
```

Comment:
This magic path does not work for every one.

Reply:
This is just for me. I don't request a pull request for this